### PR TITLE
11/27 Mobs Covid Metadata

### PR DESCRIPTION
**Pull request by jessica-davis**

### CODE REVIEW

**Comment by sbidari** [comment]
Thanks @jessica-davis for your contribution, could you include the `.gitignore` file that was removed? We also recommend including `designated_github_users` field in the metadata file to assist timely merging of PRs

**Comment by jessica-davis** [comment]
@sbidari Ok thanks, added it! I'm sorry I'm not very knowledgeable about Github. Please let me know if there is anything else I need to do for the submission.

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,5 @@ hub-config/README_files/
 /DISCLAIMER_files/
 *.html
 
-/test
\ No newline at end of file
+/test
+model-output/.DS_Store
```
Comment (sbidari) [inline .gitignore:22]:
```suggestion

```

**Review by sbidari — APPROVED** [review]
Looks good to me! Thank you for updating the metadata.

**Pull request merged**